package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "webpack serve --config webpack.dev.js  --env env=dev",
-    "gh-build-nightly": "webpack --config webpack.dev.js --env env=production basename=/website-3d-cell-viewer/",
+    "gh-build-nightly": "webpack --config webpack.dev.js --env env=production basename=/vole-app/",
     "gh-build-release": "webpack --config webpack.dev.js --env env=production basename=/website-3d-cell-viewer-release/",
```

Comment:
This remains untouched until the `website-3d-cell-viewer-release` repo is renamed (https://github.com/allen-cell-animated/website-3d-cell-viewer-release)

Reply:
oops maybe we should retire it or put a permanent redirect to vole.allencell.org